internal/plugin/resource_bundle_install_test.go

```diff
@@ -22,31 +22,32 @@ func TestAccResourceBundleInstall(t *testing.T) {
 	cfg := template.Must(template.New("enos_bundle_install").
 		Funcs(transportRenderFunc).
 		Parse(`resource "enos_bundle_install" "{{.ID.Value}}" {
-		destination = "{{ .Destination.Value }}"
+			destination = "{{ .Destination.Value }}"
```

Comment:
It doesn't matter much but the config here will render huge with all those tabs if the test fails. Generally we try and use two spaces in HCL blocks in tests.